### PR TITLE
47 oqapiattribute completeness allow selection of multiple attributes

### PR DESCRIPTION
**Pull request by mcauer**

### CODE REVIEW

File: src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <i class="close icon"></i>
+  <div class="header">
+    <div id="title">Title will be set automatically</div>
+  </div>
+  <div class="content">
+    <div class="ui three column divided grid">
+<!--      <div class="column">-->
+<!--        <h4 class="ui header">Description:</h4>-->
+<!--        <div id="description">Description text will be set automatically</div>-->
+<!--      </div>-->
```
Comment (ElJocho) [inline src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html:25]: Comment needed for future work?

**Comment by ElJocho** [comment]
Nice new feature! I added 2 suggestions to a seperate branch https://github.com/GIScience/ohsome-dashboard/pull/51 . 
The first one is making the X buttons take up more space, since with the current version I often missclicked them and opened the pop up on accident.
And another one to align the borders of the indicator containers, so that the are in line with each other.

File: src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html (hunk 1 above)
Comment (mcauer) [reply src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html:25]: Yes, it's a future implementation. The information is just not yet available in the backend

Comment (mcauer) [reply src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html:25]: Yes. We will have this in the future once the information is available in the backend

**Review by ElJocho — APPROVED** [review]

**Pull request merged**